help_cmd.go

```diff
@@ -23,17 +23,11 @@ EXAMPLES
 
 // helpExec implements the 'help' command.
 func helpExec(e *Entrypoint, args, _ []string) error {
-	if cmd, rest := e.Identify(args); IsNull(cmd) {
+	if cmd, rest, err := e.Identify(args); err != nil {
+		return err
```

Comment:
New: this can now fail (which is good! — because it gives the exoskeleton application the chance to inform the user)